tasks/lint.yaml

```diff
@@ -159,56 +159,92 @@ tasks:
   - name: license
     description: Lint for the SPDX license identifier being in source files
     actions:
-      - description: Checking for SPDX headers
+      - description: Checking for SPDX headers using YAML config with defaults
         shell:
           darwin: bash
           linux: bash
         cmd: |
-          # check for addlicense bin
-          if [ -x "$HOME/go/bin/addlicense" ]; then
-              echo "addlicense installed in $HOME/go/bin"
+          CONFIG_FILE=".license_config.yaml"
+
+          # If the config file exists, parse its values; otherwise, use default values.
+          if [ -f "$CONFIG_FILE" ]; then
+            LICENSE=$(uds zarf tools yq e '.license' "$CONFIG_FILE")
+            COPYRIGHT=$(uds zarf tools yq e '.copyright' "$CONFIG_FILE")
           else
-              echo "Error: addlicense is not installed in $HOME/go/bin" >&2
-              exit 1
+            LICENSE="AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial"
+            COPYRIGHT="Defense Unicorns"
+          fi
+
+          # Exit if license or copyright are not provided.
+          if [ -z "$LICENSE" ] || [ "$LICENSE" = "null" ] || [ -z "$COPYRIGHT" ] || [ "$COPYRIGHT" = "null" ]; then
+            echo "License or copyright configuration not provided; exiting."
+            exit 0
+          fi
```

Comment:
Should this `exit 0` if these are missing? Parsing the logic this would mean the user has a license-config file but is missing `license` and/or `copyright` keys. I would think that's likely a typo/misconfiguration and should error out?

If there's a use-case for explicitly having no license configuration I'm okay with the `null` or `none` fallbacks but should make sure that both license and copyright are set to that.